src/controller/ViewController.java

```diff
@@ -0,0 +1,72 @@
+package controller;
+
+import logic.GameProcessor;
+import gui.UserInterface;
+
+public class ViewController 
```

Comment:
Should we consider ditching this ViewController class? The idea would be for it to abstract/encapsulate the communication between the user's actions and the GameProcessor, but it seems that in most cases the ViewController won't really be encapsulating anything. The current flow would be: UI event handler -> ViewController -> GameProcessor, but the calls to GameProcessor seem like they will just be like 1 line in most cases. Should we consider just adjusting to: UI event handler -> GameProcessor, and call our event handlers the "controller" in our MVC pattern?